deployer/templates/es.yaml

```diff
@@ -92,7 +96,7 @@ objects:
                   mountPath: /etc/elasticsearch/secret
                   readOnly: true
                 - name: elasticsearch-config
-                  mountPath: /usr/share/elasticsearch/config
+                  mountPath: /etc/elasticsearch/config
```

Comment:
i like the consistency but everything is still looking in `/usr/share/elasticsearch/config` so isn't this making the configmap useless?